app/javascript/vue/protocol/modals/reorderable_items_modal.vue

```diff
@@ -24,7 +24,7 @@
                 <strong v-if="includeNumbers" class="step-element-number">{{ index + 1 }}</strong>
                 <i v-if="item.attributes.icon" class="fas" :class="item.attributes.icon"></i>
                 <span :title="item.attributes.name" v-if="item.attributes.name">{{ item.attributes.name }}</span>
-                <span v-else class="step-element-name-placeholder">{{ item.attributes.placeholder }}</span>
+                <span v-else :title="getTitle(item)">{{ getTitle(item) }}</span>
```

Comment:
We need a placeholder when the text block is empty. The name must be fixed in the serializer.

Reply:
I think we should just leave placeholder as is, and move the getTitle(item) thing to the 'name' line above it. If, after removing images and tables, the title is empty, the placeholder will kick in.